src/index.js

```diff
@@ -1,4 +1,4 @@
-/* global fetch */
+import 'isomorphic-fetch';
 import { isEqual } from 'lodash';
```

Comment:
Since this is a library, I would encourage you to eventually remove lodash and only import the bits of lodash you need like `lodash.isequal` package. Lodash is large, and this will reduce the bundle size.